src/redux/rockets/RocketAPI.js

```diff
@@ -0,0 +1,19 @@
+import axios from 'axios';
+import { apiState, rocketData } from './RocketSlice';
+
+const rocketApi = 'https://api.spacexdata.com/v4/rockets';
+
+const getRockets = () => (dispatch) => {
+  dispatch(apiState('pending'));
+
+  axios.get(rocketApi)
+    .then((response) => {
+      dispatch(apiState('success'));
+      dispatch(rocketData(response.data));
+    })
+    .catch((error) => {
+      dispatch(apiState(error));
+    });
+};
+
+export default getRockets;
```

Comment:
- [ ] it would be better to fetch the data from the `API` just once so that you avoid redundant fetchint that will make bigger apps slower so please try to do that as [required](https://github.com/microverseinc/curriculum-react-redux/blob/ac8c5ee08c46b1cef0a47e2e4f9adad1d26a5c00/group-project/project_space_travelers_hub.md?plain=1#L97)
![Screenshot 2023-04-13 at 17 49 50](https://user-images.githubusercontent.com/61790587/231843126-b5d1cb1b-841d-4c9d-9ca1-1042872a148a.png)